store/map.js

```diff
@@ -105,19 +118,17 @@ export const actions = {
   // }),
 };
 
-export const getters = {
-
-};
+export const getters = {};
 
 // helpers
-const getFileName = ({name, fileName}) => {
-  return `${name}-${dateTimeStamp()}-${fileName}`
-}
+const getFileName = ({ name, fileName }) => {
```

Comment:
make these standalone functions + import them.

If it was needed here, you'll likely use it somewhere else at somepoint.


```
// new file, functions/getFileName.js
const getFileName = ({ name, fileName }) => {
  return `${name}-${dateTimeStamp()}-${fileName}`;
};
export default getFileName;


// here
import getFileName from '~/functions/getFileName';
```

And you can use it project wide when needed

Reply:
(and clean up this file)

Reply:
yeah seriously omg lol